.github/workflows/review.yml

```diff
@@ -72,15 +72,25 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v3
 
-      - name: Build container image
-        uses: "./.github/actions/build-container"
+      - name: Calculate environment variables
+        shell: sh
+        run: echo "SHORT_SHA=`git rev-parse --short HEAD`" >> $GITHUB_ENV
+
+      - name: Login to GCP
+        id: gcloud
+        uses: elgohr/gcloud-login-action@v1 #TODO - v2 is bugged, unable to get outputs
```

Comment:
Why are you using an action from elgohr?
Are not actions published by Google?
In this workflow we used some of them for container registry https://github.com/hoprnet/notification-service/blob/master/.github/workflows/ci-publish-docker.yml ?